src/components/navigation/VerticalNavigation.vue

```diff
@@ -26,4 +26,8 @@ defineProps<Props>()
     text-align: start;
     padding-left: 5%;
 }
+
```

Comment:
Right now, the presence of the navigation on the right is a bit "too prominent". Maybe we can change this such that the font is a bit smaller and that not all capitals are used for the navigation?

Would it be possible to change this component such that navigation items with children look different from the other items? (In the original website, "Case studies" is for example a component with some child-navigation links, I think we should include these to). 

<img width="238" alt="image" src="https://user-images.githubusercontent.com/9608686/187159515-10cc08b6-50b8-4caf-971c-b9609692e197.png">